packages/fxa-settings/src/components/ConfirmWithLink/index.stories.tsx

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ConfirmWithLink from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { DefaultSubject, SubjectCanGoBack, SubjectWithWebmail } from './mocks';
+
+export default {
+  title: 'components/ConfirmWithLink',
+  component: ConfirmWithLink,
+} as Meta;
+
+export const Default = () => (
+  <LocationProvider>
+    <AppLayout>
+      <DefaultSubject />
+    </AppLayout>
+  </LocationProvider>
+);
+
+export const UserCanGoBack = () => (
+  <LocationProvider>
+    <AppLayout>
+      <SubjectCanGoBack />
+    </AppLayout>
+  </LocationProvider>
+);
+
+export const withWebmailLink = () => (
```

Comment:
~withWebmailLink~ WithWebmailLink

Reply:
Re: mockCallback - `afterEach` should do the trick, no? I did console log the calls and it reset in between tests :)

Reply:
Intresting... https://github.com/storybookjs/eslint-plugin-storybook/blob/main/docs/rules/prefer-pascal-case.md might also do the trick if we want to enforce PascalCase for export naming.